src/cluster/pager.rs

```diff
@@ -43,6 +44,7 @@ impl<
         &'a mut self,
         query: Q,
         state: PagerState,
+        qp: QueryParams,
```

Comment:
Seems it's going to be a breaking change which seems to be unnecessary.
Can you please introduce a new `Pager` method like `query_with_pager_state_params`. I know it_looks_a_bit_too_long but will help to keep it back compatible

Reply:
That is a good idea